### DISCUSSION TITLE
This project is amazing, when can I use it in production。wait v1.0.0

### DISCUSSION THREAD

**Discussion by DreamPWJ** [arthelokyo/astrowind#186] [Q&A]
I found that there are several branches now, and they are still iterating frequently

**Comment by prototypa** [comment]
Hi @DreamPWJ It's great that you like the project. We're very pleased

It's a good question you just asked: **When can I use it in production?**

We certainly need to better document and organize the process. At the moment we have 3 branches.

[**main**](https://github.com/onwidget/astrowind/tree/main): Here is a stable code from the project using the _Astro 2.0_ version. At any time you can make yours from the green button or doing a fork. We try that this code in main is always stable and any complicated issue that appears we usually fix it there. Usually the changes there are library updates and fixes.

[**0.x**](https://github.com/onwidget/astrowind/tree/0.x): Here we have the previous stable version of the template using _Astro 1.0_. Same thing, it's a stable code that we don't usually change.

From any of these branches, the code is usually stable and complies with the reported functionalities and characteristics, such as great performance, good SEO, etc. It has been tested by a good community of users and you can take it as a start for your project.

However, the project is not yet in a stable 1.0 version in the sense that if you fork, the API it may suffer some breaking changes. If you don't update it doesn't affect you and you will have a stable project to continue your work, but if you want to take frequent updates that can be a little problem until we release 1.0.

We decided to start this template and others that we are developing (for example for _Qwik_ and _Next JS_, even less advanced), because for a long time we used external resources every time we needed to start a project and at some point we realized that we were losing a lot of time adapting other things instead of having our own templates well adapted to the latest standards, with great performance and SEO (which is something that we were constantly fixing in external projects, it is not something very usual). And of course we decided to open source them to benefit others and ourselves from the collective experience.

We mention this to you because we are also interested in making this code very stable, knowable and easy to update. Unfortunately due to the nature of Astro or Tailwind CSS we have not yet found an easy way to do many of these things as reusable and installable libraries, although we have it as something to assess to find a viable solution.

[**ui**](https://github.com/onwidget/astrowind/tree/ui): We are currently working on the third branch **ui** to release **AstroWind** version 1.0. There are some small breaking changes and once released we would only try to upload new features or fix problems that are reported without touching much of that code so that it is easy for us or any user to make specific fixes.

We had planned to deliver **AstroWind** 1.0 at the end of the previous month but we have moved the date to mid-February. Other responsibilities occupy us.

Soon we will elaborate a more detailed guide of what comes with **AstroWind** 1.0 and we will have a process to test it well and receive feedback.

Thank you, anything do not hesitate to write to us.

**Comment by DreamPWJ** [comment]
Thank you very much for your serious answer, I will use it immediately in 1.0.0, and it will be better if it can be better upgraded and a new UI design later

**Comment by DreamPWJ** [comment]
algolia support？

**Comment by DreamPWJ** [comment]
i18n support ？@prototypa

**Comment by prototypa** [comment]
Hi @DreamPWJ 

We are taking into account how to make easy to apply i18n, it will not be included by default. Implementing Angolia or some other search service I think is something that can already be achieved now by creating some script and a small component but it will not be part of the code. Once version 1.0.0 comes out we plan to provide examples for each of the most important specific situations like this i18n or Angolia.

**Comment by DreamPWJ** [comment]
ant design UI  support ？
